### PR TITLE
Add Q^ω, as a better distinguishment of Erdős space

### PR DESCRIPTION
**Pull request by yhx-12243**
This is a continuation of #1141/#1142.

* The arguments of LOTS may tedious, it is possible to open an MSE post.
* Also Add the argument that $\mathbb R^\omega$ is not Menger.

### CODE REVIEW

File: spaces/S000146/properties/P000050.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000050.md:9]:
```suggestion
The product of {P50} spaces is {P50}, so this follows from {S27|P50}.
```

File: spaces/S000146/properties/P000053.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000053.md:7]: This will follow from LOTS+second countable.

File: spaces/S000146/properties/P000056.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000056.md:7]:
```suggestion
Let $F_q = \{f\in\mathbb Q^\omega:f(0)=q\}$, then $F_q$ is a closed set with empty interior, and $\mathbb Q^\omega = \bigcup_{q \in \mathbb Q} F_q$.
```

File: spaces/S000146/properties/P000133.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000133.md:10]:
```suggestion
In fact, {S146} can be viewed as a subspace of $\omega^\omega$ which $\lim_{n \to + \infty} x_n = \infty$, by {{zb:0642.54033}} (<https://eudml.org/doc/220890>).
```

File: spaces/S000146/properties/P000133.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000133.md:10]:
```suggestion
In fact, {S146} can be viewed as a subspace of $\omega^\omega$ which $\lim_{n \to + \infty} x_n = \infty$, by Theorem 2.7 of {{zb:0642.54033}} (<https://eudml.org/doc/220890>).
```

**Comment by yhx-12243** [comment]
I'm not sure that we added LOTS currently because it may be not so obvious, or if we can make a mathse post.
The reason I keep metrizable is similar because it is much simpler than the argument of LOTS.

(Another offtopic: $\mathbb Q^\omega$ and $\mathbb Q \times \mathbb P$ are pi-base-indistinguishable)

File: spaces/S000146/properties/P000133.md
Comment (StevenClontz) [inline spaces/S000146/properties/P000133.md:12]:
```suggestion
The result then follows as $\omega^\omega$ is homeomorphic to {S28}, and {S28|P133}.
```

Comment (StevenClontz) [reply spaces/S000146/properties/P000133.md:12]: Oops, hang on; this is only enough for GO-space...

Comment (yhx-12243) [reply spaces/S000146/properties/P000133.md:12]: Well because {P133} (LOTS) is not directly hereditarily, my argument is that:
If $A$ is embeddable into $\mathbb R$ and the embedding is dense in $\mathbb R$, then $A$ is LOTS.
Otherwise it is not always true, e.g., $A = (0, 1) \cup \{2\}$, it is the counterexample of {S198}.

So it is not so simple and that's the reason I hanged.

Comment (StevenClontz) [reply spaces/S000146/properties/P000133.md:12]: I think I agree with you. I couldn't find any proof that a dense subspace of a LOTS is LOTS on MSE, so that's a good question to ask and answer there. So I think we should remove this assertion for now, and it can be added after a satisfactory MSE post.

**Review by StevenClontz — CHANGES_REQUESTED** [review]
Let's defer the proof for LOTS for after a discussion on MSE.

**Comment by yhx-12243** [comment]
> Let's defer the proof for LOTS for after a discussion on MSE.

Agree. I have not found any post that "dense subspace of a LOTS is dense" and I think it's right (it is not hard to prove), I worry that whether I make something wrong. 🤐

**Comment by prabau** [comment]
The two aliases for S146 don't seem to serve much purpose.  Unless we can justify their existence here, they should be removed.

File: spaces/S000146/properties/P000066.md
Comment (prabau) [inline spaces/S000146/properties/P000066.md:8]:
```suggestion
It follows from {S28|P66} and the fact that {P66} is preserved by closed subspaces.
```

File: spaces/S000030/properties/P000066.md
Comment (prabau) [inline spaces/S000030/properties/P000066.md:8]:
```suggestion
It follows from {S28|P66} and the fact that {P66} is preserved by closed subspaces.
```

**Comment by prabau** [comment]
Since this PR used twice the fact that closed subspaces of Menger spaces are Menger, it would be good to have this as a meta-property.  But it can be done later.

**Comment by prabau** [comment]
Actually, re-reading what is there to justify P66, what is written there makes little sense.  The second sentence starts with "It follows from". That seems to mean the "It" refers to the previous sentence, which is not what you mean at all.

Let me rephrase the whole thing for one of them, and you can do the other one.

File: spaces/S000030/properties/P000066.md
Comment (prabau) [inline spaces/S000030/properties/P000066.md:8]:
```suggestion
And {P66} is preserved by closed subspaces.
But {S28|P66}.
```

**Comment by prabau** [comment]
In the description for the new space, shouldn't we mention it's a subspace of $\mathbb R^\omega$ (S30)?

**Comment by prabau** [comment]
`refs:` section for S146 (and one other place): When referencing a paper from the literature or a book, we usually add the name of the author/authors in parentheses after the title, for the `name:` field.

See for example https://topology.pi-base.org/theorems/T000277/references

**Comment by prabau** [comment]
S146 - P53: The English again makes no sense.  _Please_, fix the grammar.

**Comment by prabau** [comment]
S146 - P27: same thing.  Fix the grammar.

**Comment by prabau** [comment]
P133: I agree that the LOTS is too much and should be a post on mathse.

File: spaces/S000146/properties/P000087.md
Comment (prabau) [inline spaces/S000146/properties/P000087.md:7]:
```suggestion
This space is an additive subgroup of {S30}, which has a group topology.
```

**Comment by prabau** [comment]
Sorry, but the grammar for P27, P53 still does not make sense.  Do you see why?

**Comment by prabau** [comment]
Finally! Thank you :)

**Comment by prabau** [comment]
Apart from removing the trait for LOTS, these were all my comments.

**Comment by yhx-12243** [comment]
> > Let's defer the proof for LOTS for after a discussion on MSE.
> 
> Agree. I have not found any post that "dense subspace of a LOTS is again a LOTS" and I think it's right (it is not hard to prove), I worry that whether I make something wrong. 🤐

In fact the proposition is that "A [dense](https://en.wikipedia.org/wiki/Dense_set) subspace of a [dense](https://en.wikipedia.org/wiki/Dense_order) LOTS is a LOTS", where two dense has different meaning.

File: spaces/S000146/README.md
Comment (prabau) [inline spaces/S000146/README.md:6]:
```suggestion
```
Does not seem useful here

File: spaces/S000146/README.md
Comment (prabau) [inline spaces/S000146/README.md:13]:
```suggestion
    name: Characterizations of the countable infinite product of rationals and some related problems (van Engelen)
  - zb: "0562.54054"
    name: Countable products of zero-dimensional absolute $F_{\sigma \delta}$ spaces (van Engelen)
```

File: spaces/S000146/README.md
Comment (prabau) [inline spaces/S000146/README.md:18]:
```suggestion
This space is a subspace of {S30}.
Note that it is not homeomorphic to {S142}.
```

**Review by prabau — APPROVED** [review]

**Comment by prabau** [comment]
Approved, but @StevenClontz needs approval too to unblock.

**Review by StevenClontz — APPROVED** [review]
Marking approved per @prabau review

**Pull request merged**
